src/config.json

```diff
@@ -74,6 +74,12 @@
             "text": "View Katib Experiments",
             "desc": "Katib",
             "link": "/katib/"
+        },
+        {
+          "type": "item",
```

Comment:
If we want to display this in the sidebar menu, it should be added to `menuLinks`.

Reply:
You are completely right, fixed it